pkg/clients/clients.go

```diff
@@ -126,6 +128,9 @@ func New(cc *controllercmd.ControllerContext) (*Clients, error) {
 }
 
 func (c *Clients) StartInformers(ctx context.Context) {
+	if c.FeatureGatesAccessor != nil {
+		go c.FeatureGatesAccessor.Run(ctx)
+	}
```

Comment:
If `c.FeatureGatesAccessor` is `nil` here, that's a programmer error right? If so, seems reasonable to let a nil pointer error panic happen like we would with any of the other struct fields?